my-vue3-app/src/components/ReportList.vue

```diff
@@ -32,6 +34,11 @@ export default defineComponent({
   components: {
     SingleReport,
   },
+  data() {
+    return {
+      nick: '',
```

Comment:
So it means the function which triggered on nick input should first of all commit in store the filters nick state change, so then it can be showed on the front in the message?

Reply:
It is a little harder :)
1. We store it in pagination when response comes
2. We may use its new value before storing FOR calculation `viewIds`
3. In the future we may want to have *dirty* value and *clean* one. 

Explanation for *dirty*. We have input with "warsawa", we have pagination with "warsawa", we add '1' char to filter, but we want wait 100ms before send filter request, so we have 2 different values "warsawa1" and "warsawa". In these 100ms user may add or remove some characters, and we still need to wait.